### PR TITLE
finished assignment

### PR DESCRIPTION
**Pull request by luke-dowling**

### CODE REVIEW

File: luke/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// task 1
+let snackServings = 26;
+let guests = 9;
+
```
Comment (ciamac-da) [inline luke/index.js:5]: You forgot Add snackServings and guests, and print the result to the console
console.log(snackServings + guests) could be the solution!

File: luke/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+let monkeys = 15;
+let bananas = 9;
+console.log(monkeys / bananas);
+
+// task 6
+let monkeyFights = 20;
+let resultTwo = (monkeys + bananas) * monkeyFights;
+console.log(resultTwo);
+
+// task 7
```
Comment (ciamac-da) [inline luke/index.js:31]: Your answers about 8. and 9. are wrong. You can increment and decrement the variables like this :
8.  console.log(++monkeys);
9.  console.log(--bananas);

**Review by ciamac-da — CHANGES_REQUESTED** [review]
Thanks a lot for pushing you Code Luke! Please read my comments, that i wrote and try to fix the code and push again.

File: luke/index.js (hunk 1 above)
Comment (luke-dowling) [reply luke/index.js:5]: sorry i must have not read that properly

File: luke/index.js (hunk 2 above)
Comment (luke-dowling) [reply luke/index.js:31]: yeah i looked this up today actually and yeah have changed that and pushing now, thanks ciamac!

Comment (ciamac-da) [reply luke/index.js:31]: You're welcome:)

**Review by ciamac-da — APPROVED** [review]
🥇

**Pull request merged**